IBAnimatable/MaskType.swift

```diff
@@ -46,23 +44,23 @@ public extension MaskType {
       return
     }
     
-    let nameAndParames = MaskType.extractNameAndParams(from: string)
+    let nameAndParames = MaskType.extractNameAndParams(from: string.lowercased())
```

Comment:
Yes, actually, didn't know if this will be our case-insensitive strategy (Need that all enums raw values are lowercased). But I didn't find anything better.